source/linear-algebra/source/02-EV/07.ptx

```diff
@@ -128,14 +128,74 @@ Rewrite this solution space in the form <me>\setBuilder{ a \left[\begin{array}{c
 </task>
 <task>
     <p>
-Rewrite this solution space in the form <me>\vspan\left\{\left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown\end{array}\right], \left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown \end{array}\right]\right\}.</me>
+Which of these choices best describes the set of two vectors
+<m>\left\{\left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown\end{array}\right], \left[\begin{array}{c} \unknown \\ \unknown \\ \unknown \\ \unknown \end{array}\right]\right\}</m>
+used in this solution space?
+<ol marker="A.">
+<li>
+    <p>
+    The set is linearly dependent.
+    </p>
+  </li>
+<li>
+    <p>
+        The set is linearly independent.
+    </p>
+  </li>
+  <li>
+      <p>
+        The set spans the solution space.
+      </p>
+    </li>
+  <li>
+    <p>
+        The set is a basis of the solution space. 
+    </p>
+  </li>
+</ol>
+    </p>
+</task>
+</activity>
+<answer>
+    <p>
+        D.
+    </p>
+</answer>
```

Comment:
The above `<task>` needs a `<statement>`, and then this `<answer>` should be a sibling of that statement.